src/briefcase/console.py

```diff
@@ -255,8 +255,6 @@ def _build_log(self, command):
                 for func in self.log_file_extras:
                     try:
                         func()
-                    except KeyboardInterrupt:
-                        raise
                     except Exception:
                         self.error(traceback.format_exc())
```

Comment:
Unless I'm confused, there is no need to re-raise; `KeyboardInterrupt` will propagate naturally.

Reply:
AFAICT, that's the whole reason they derived both `Exception` and [`KeyboardInterrupt`](https://docs.python.org/3/library/exceptions.html#KeyboardInterrupt) from `BaseException`....so you didn't have to worry about `except Exception` catching `KeyboardInterrupt`. IIRC, this was a Python 2->3 change.

I can definitely add a test for it, though.

Reply:
Huh... you are absolutely correct. TIL another thing that I need to forget after the Py2->3 transition :-)

Regardless, a test can't hurt.

Reply:
Looks like [test_save_log_to_file_extra_interrupted](https://github.com/rmartin16/briefcase/blob/main/tests/console/test_Log.py#L147) already exists and ensures the correct behavior.